lang/src/org/partiql/lang/domains/partiql-domains.kt

```diff
@@ -42,6 +42,7 @@ class PartiqlAst private constructor() {
             second: Long,
             nano: Long,
             precision: Long,
+            withTimeZone: Boolean,
```

Comment:
Hmm, the `partiql-domains.kt` shouldn't be checked into the repo. From the `.gitignore`: https://github.com/partiql/partiql-lang-kotlin/blob/06a2a5df5fe040dae1a3b236ccda954453ce1c41/lang/src/org/partiql/lang/domains/.gitignore#L1-L2

Reply:
This is clearly not working.  On the other hand, it looks like we to remove the file completely.

Reply:
Ok [this](https://stackoverflow.com/questions/49520447/gitignore-not-working-for-nested-files) helped, did the following:
1. Remove the file.
2. Commit the change.
3. re-build to ensure file is created.
4. `git status` to ensure file will not get included.

Reply:
Thanks for sorting it out! Looks like the merge PR #517 had inadvertently added it.